src/mpid/ch4/netmod/ofi/ofi_progress.h

```diff
@@ -94,6 +94,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_progress(int vci, int *made_progress)
     } else if (likely(1)) {
         for (int nic = 0; nic < MPIDI_OFI_global.num_nics; nic++) {
             int ctx_idx = MPIDI_OFI_get_ctx_index(vci, nic);
+            if (MPIDI_OFI_global.ctx[ctx_idx].cq == NULL)
+                continue;
```

Comment:
Add a comment on its reason (that not all cq is initialized during init) and a FIXME on better multi-nic progression.

Reply:
done